.gitignore

```diff
@@ -57,4 +57,7 @@ examples/int8/qat/qat
 examples/int8/training/vgg16/data/*
 examples/int8/datasets/data/*
 env/**/*
-bazel-Torch-TensorRT-Preview
\ No newline at end of file
+bazel-Torch-TensorRT-Preview
```

Comment:
We don't need this anymore. Can we get rid of this?

Reply:
I dont think it hurts to keep it around just in case